meta/update_bootstrap.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+: "${CURRENT_JAKT_COMPILER:=build/jakt}"
+: "${JAKT_RUNTIME_DIR:=build/runtime}"
```

Comment:
this script assumes the selfhost built jakt rather than cmake? intended or an artifact of 28 commits?

Reply:
intended, but I'll change the default.